.github/workflows/reusable-release.yml

```diff
@@ -115,7 +121,7 @@ jobs:
       - uses: actions/setup-java@v4 # don't 'cache: sbt', at least until https://github.com/actions/setup-java/pull/564 is merged
         with:
           distribution: corretto
-          java-version: 17
+          java-version-file: ${{ inputs.LIBRARY_BUILD_JAVA_VERSION_FILE }}
```

Comment:
When we run the library's build process, let's respect the `.tools-version` that they supply - see https://github.com/actions/setup-java/pull/606